runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -374,6 +382,7 @@ impl<T: Config> Pallet<T> {
 			disputed_bitfield
 		};
 
+		bitfields_processed_metric.inc_by(signed_bitfields.len().try_into().unwrap_or_default());
```

Comment:
I don't get the `try_into().unwrap_or_default()`. Shouldn't this be a `let _ = ` to silence the unused results?

Reply:
There is no unused result in this case, it's converting the value to u64.